lib/credentials/credential_provider_chain.js

```diff
@@ -154,6 +154,7 @@ AWS.CredentialProviderChain = AWS.util.inherit(AWS.Credentials, {
  *   function () { return new AWS.EnvironmentCredentials('AMAZON'); },
  *   function () { return new AWS.SharedIniFileCredentials(); },
  *   function () { return new AWS.ECSCredentials(); },
+ *   function () { return new AWS.SsoCredentials(); },
```

Comment:
SsoCredentials get preference over SharedIni
```suggestion
 *   function () { return new AWS.SsoCredentials(); },
 *   function () { return new AWS.SharedIniFileCredentials(); },
 *   function () { return new AWS.ECSCredentials(); },
```